packages/fields/src/fields/password/index.tsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * WordPress dependencies
+ */
+import type { Field } from '@wordpress/dataviews';
+
+/**
+ * Internal dependencies
+ */
+import type { BasePost } from '../../types';
+import PasswordEdit from './edit';
+
+const passwordField: Field< BasePost > = {
+	id: 'password',
+	type: 'text',
+	getValue: ( { item } ) => item.password,
+	Edit: PasswordEdit,
+	enableSorting: false,
+	enableHiding: false,
+	isVisible: ( item ) => item.status !== 'private',
+};
+
+export default passwordField;
```

Comment:
Could you add a bit of JSDoc for this export, so it shows up in the docs (README) instead of "undocumented declaration"? (see parentField example)

Reply:
Yup will do, one minute :)

Reply:
Done: https://github.com/WordPress/gutenberg/pull/66567/commits/6abc19e492b8b778be8a301d7798e51fe8f47b2e